app/views/homepage/_references.html.erb

```diff
@@ -0,0 +1,101 @@
+<%
+  references = [
+    {
+      title: "Pipeline",
+      links: [
+        {
+          title: "Step types",
+          link: "/docs/pipelines/command-step"
```

Comment:
If you merge main, you should be able to link to the overview: `/docs/pipelines/step-reference`